### PR TITLE
feat: add gravity chain

### PR DESCRIPTION
**Pull request by myz1237**

### CODE REVIEW

**Comment by myz1237** [comment]
blockchair explorer is invalid now, shall we change to okx explorer? @chybisov

**Comment by chybisov** [comment]
> blockchair explorer is invalid now, shall we change to okx explorer? @chybisov

Why do you think so? Blockchair works fine. Sometimes bitcoin tests just take more time to execute. Please rollback changes.

**Comment by myz1237** [comment]
But it usually fails, blocking the next test @chybisov

**Comment by myz1237** [comment]
Can you access this page? https://blockchair.com/bitcoin/

**Comment by chybisov** [comment]
> Can you access this page? https://blockchair.com/bitcoin/

Yes, sometimes they fail. Those tests do not always show the real picture, it is more like advice to check if the link is still alive. We can try find more reliable explorer, but it should be a good one, not some CEX like okx.

image

File: src/chains/supportedChains.evm.ts [hunk 1]
```diff
@@ -927,4 +927,28 @@ export const supportedEVMChains: EVMChain[] = [
...
+    id: 1625,
+    mainnet: true,
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/chains/gravity.svg',
+    multicallAddress: multicallAddresses[ChainId.GRA],
+    metamask: {
+      chainId: prefixChainId(1625),
+      blockExplorerUrls: ['https://explorer.gravity.xyz/'],
+      chainName: 'Gravity Alpha Mainnet',
+      nativeCurrency: {
```
Comment (myz1237) [inline src/chains/supportedChains.evm.ts:946]: This part is from mm

**Review by chybisov — APPROVED** [review]

**Pull request merged**